### DISCUSSION TITLE
[Suggestion/Request?] Multiplayer (And Sp) spawn select

### DISCUSSION THREAD

**Discussion by Jorpis-Jorah** [tsunamayo/Starship-EVO#4148] [Suggestions]
Upon the update for Mp framework, I have a request. It's pretty simple, Upon joining or creating a new world, A small menu to pick where to spawn in the system. It would be like the teleport menu in the codex, you could select an entity, whether a station or ship, and spawn. Simple!

**Comment by tsunamayo** [comment]
Yes I agree this is a must have.
